ci/prow/e2e_prebuilt_kernel_module

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
```

Comment:
What is the difference between that and `#!/usr/bin/bash`?

Reply:
https://stackoverflow.com/a/16365367